app/models/services/locators/doi_paper_locator.rb

```diff
@@ -15,6 +15,7 @@ def find_or_import_paper
 
     if paper_attributes
       response = crossref.resource.response
+      (paper_attributes[:links_attributes].map {|l| l[:link_editable] = false}) if paper_attributes[:links_attributes].present?
```

Comment:
I was going to do something similar to turn the validations on and off with a virtual attribute (like `locator_type` and `locator_id` on `Paper`) called `imported`. If `imported` was true, I was going to turn off the validations or something like that. I can't remember exactly what my plan was, now.